wagtail/wagtailcore/models.py

```diff
@@ -50,13 +50,14 @@ def get_by_natural_key(self, hostname, port):
 
 @python_2_unicode_compatible
 class Site(models.Model):
-    hostname = models.CharField(verbose_name=_('Hostname'), max_length=255, db_index=True)
-    port = models.IntegerField(verbose_name=_('Port'), default=80, help_text=_("Set this to something other than 80 if you need a specific port number to appear in URLs (e.g. development on port 8000). Does not affect request handling (so port forwarding still works)."))
-    root_page = models.ForeignKey('Page', verbose_name=_('Root page'), related_name='sites_rooted_here')
-    is_default_site = models.BooleanField(verbose_name=_('Is default site'), default=False, help_text=_("If true, this site will handle requests for all other hostnames that do not have a site entry of their own"))
+    hostname = models.CharField(verbose_name=_(u'Hostname'), max_length=255, db_index=True)
```

Comment:
Please don't add ``u`` prefixes to strings. All modules have the ``unicode_literals`` future import at the top which does this automatically.